src/gatling/simulations/uk/gov/hmcts/reform/exui/performance/scenarios/EXUIIACMC.scala

```diff
@@ -432,10 +520,11 @@ object EXUIIACMC {
 
     .pause(MinThinkTime , MaxThinkTime)
 
-    .group("XUI${service}_270_ShareACaseConfirm") {
-      exec(http("XUI${service}_270_ShareACaseAssignments")
+    .group("XUI${service}_280_ShareACaseConfirm") {
+      exec(http("XUI${service}_280_ShareACaseAssignments")
         .post("/api/caseshare/case-assignments")
-        .headers(IACHeader.headers_userassignment)
+        .headers(Headers.commonHeader)
+        .header("accept", "application/json, text/plain, */*")
         .body(ElFileBody("bodies/iac/IACShareACase.json")).asJson
```

Comment:
Remove the .asJson function, as this will overwrite the accept header on the previous line.